l2-contracts/contracts/ConsensusRegistry.sol

```diff
@@ -0,0 +1,483 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.20;
+
+import {Ownable2Step} from "@openzeppelin/contracts/access/Ownable2Step.sol";
```

Comment:
I still want to make this contract upgradable, in that case upgradeable version of the contract should be used.
```suggestion
import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
```